src/cdboost-devstone-atomic.hpp

```diff
@@ -66,7 +64,7 @@ class PDEVStoneAtomic : public boost::simulation::pdevs::atomic<TIME, MSG>
      * @brief internal function.
      */
     void internal() noexcept {
-        dhrystoneRun (_internal_cycles);
+        DhryStone().dhrystoneRun(_internal_cycles);
```

Comment:
In general we do not use CamelCase, we try to follow STL/Boost code standard. Since we use those libraries a lot, we follow they standard to keep it consistent.